### DISCUSSION TITLE
Limit battery charge threshold on laptop while connected to power

### DISCUSSION THREAD

**Discussion by mickystevo85** [linuxdeepin/developer-center#2957] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hi all, just raising an issue/change that I hope can be implemented. 

I'm hoping that a limit could be set when leaving a laptop on charge, to prolong battery health.

As I leave my Huawei Matebook 14 2020 AMD edition on charge most of the time and I would love to set the maximum limit to 60% or 80%.

**Comment by meiyixiang** [comment]
It's probably hardware related, it's hard to control the software level so that it can only charge 60 or 80 percent
